### PR TITLE
Fix collapsible card not expanding

### PR DESCRIPTION
**Pull request by MattLichtenstein**
## Description

- This fixes the issue where collapsible card would not expand again after being collapsed

## Reference

[VPN-2033: Multi-hop collapsible card can only be expanded once](https://mozilla-hub.atlassian.net/browse/VPN-2033) / #3267

[VPN-923: Add accessibleName to VPNCollapsibleCard toggle](https://mozilla-hub.atlassian.net/browse/VPN-923) /  #1665

## Checklist
    
- [x] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [x] I have performed a self review of my own code
- [x] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

File: translations/strings.yaml [hunk 1]
```diff
@@ -563,3 +563,11 @@ freeTrials:
     value: Sign up to start your 7-day free trial.
     comment: This string appears above the 'Sign up' button on mobile onboarding views
   freeTrialLabel: 7-day free trial
+
+general:
+  expandCard:
+    value: Expand card
```
Comment (lesleyjanenorton) [inline translations/strings.yaml:569]: @flodolo Do you know if there are any conventions for this sort of "See more" / "See less" language in Firefox or Moz.org?

File: nebula/ui/components/VPNCollapsibleCard.qml [hunk 2]
```diff
@@ -154,18 +167,14 @@ Rectangle {
...
                 width: parent.width
 
-                Behavior on height {
-                     NumberAnimation {
-                         duration: animationDuration
-                         easing.type: Easing.OutQuad
-                     }
-                }
+                //becomes invisible so that it is no longer calculated in the implicitHeight (or picked up by screen readers)
+                visible: opacity > 0
```
Comment (lesleyjanenorton) [inline nebula/ui/components/VPNCollapsibleCard.qml:171]: Nice!

**Review by lesleyjanenorton — APPROVED** [review]

File: translations/strings.yaml (hunk 1 above)
Comment (flodolo) [reply translations/strings.yaml:569]: Normally I see [expand](https://transvision.flod.org/?recherche=expand&repo=gecko_strings&sourcelocale=en-US&locale=en-US&search_type=strings_entities) and [collapse](https://transvision.flod.org/?recherche=collapse&repo=gecko_strings&sourcelocale=en-US&locale=en-US&search_type=strings_entities).

**Review by flodolo — APPROVED** [review]

**Review by flozia — APPROVED** [review]

**Comment by MattLichtenstein** [comment]
@flodolo Would you mind reviewing the last commit (change to strings.yaml) as a sanity check. Thanks!

**Pull request merged**